### PR TITLE
Adding runtime styling info

### LINKED ISSUE #707: Create new page on runtime styling
We could stand to have a devoted page about runtime styling. i.e. how to get a map layer and change its properties.

Related: https://www.mapbox.com/help/android-runtime-styling-intro/

### PR DESCRIPTION
**Pull request by langsmith**
This pr resolves #707 by adding a good amount of runtime styling info to https://www.mapbox.com/android-docs/maps/overview/styling-map/, rather than creating a completely new page for runtime styling. 

screencapture-localhost-8080-android-docs-maps-overview-styling-map-2018-11-30-10_08_22

### CODE REVIEW

File: src/pages/maps/overview/styling-map.md [hunk 1]
```diff
@@ -1,13 +1,128 @@
...
 title: "Styling the map"
 description: "Learn how to use the Mapbox Maps SDK for Android to change your map style to a custom style or a pre-made style made by the talented Mapbox cartography team."
+prependJs:
+  - "import CodeLanguageToggle from '../../../components/code-language-toggle';"
+  - "import ToggleableCodeBlock from '../../../components/toggleable-code-block';"
+
 ---
 
-The Maps SDK for Android allows full customization when it comes to displaying your map. This means you can brand the map with the colors, icons, and fonts that match your apps UI. The customization can either be done using [Mapbox Studio](https://www.mapbox.com/help/studio-manual/) or during runtime using the [Runtime Styling APIs](https://www.mapbox.com/android-docs/map-sdk/overview/runtime-styling/).
+The Maps SDK for Android allows full customization of the look of the map. The map's colors, icons, and fonts can be adjusted to match the app's UI and your company's branding. Customization can be done via the browser-based [Mapbox Studio](https://www.mapbox.com/help/studio-manual/) or with the Maps SDK's runtime styling.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/styling-map.md]: https://www.mapbox.com/help/studio-manual/ --> https://www.mapbox.com/studio-manual/

(just avoids a redirect)

Comment (colleenmcginnis) [inline src/pages/maps/overview/styling-map.md]: >The Maps SDK for Android allows full customization of the look of the map. The map's colors, icons, and fonts can be adjusted to match the app's UI and your company's branding. Customization can be done via the browser-based [Mapbox Studio](https://www.mapbox.com/help/studio-manual/) or with the Maps SDK's runtime styling. 

reword... 

>The Maps SDK for Android allows full customization of the look of the map. You can choose one of Mapbox's core styles designed by our cartographers or create a custom map style by adjusting the map's colors, icons, and fonts to match your application's UI or company's brand. 
>
>There are two approaches to customizing the look of the map: creating a custom map style with Mapbox Studio and loading it into your application or updating map features at runtime using the Maps SDK for Android.

File: src/pages/maps/overview/data-driven-styling.md [hunk 2]
```diff
@@ -650,4 +650,4 @@ geoJsonSource?.setGeoJson(featureCollection)
 
 ## Capturing click events
 
-Layers are not clickable and don't expose any event listeners for you to handle user input. Instead, the map querying feature described in a separate doc goes over how to detect when a user has clicked on a polygon inside your fill layer for example.
\ No newline at end of file
+Layers are not clickable and don't expose any event listeners for you to handle user input. [The Mapbox Annotation Plugin](/android-docs/plugins/overview/annotation/#manager) provides `onClick()` and `onLongClick()` listening though. Instead, the [map querying](/android-docs/maps/overview/query/) tools can help you detect when a user has interacted with the map. For example, when a `FillLayer`'s `Polygon` has been tapped on.
```
[the remark this answers is not in the document]
Comment (langsmith) [reply src/pages/maps/overview/data-driven-styling.md]: Still true. Switched to your version

File: src/pages/maps/overview/styling-map.md (hunk 1 above)
Comment (langsmith) [reply src/pages/maps/overview/styling-map.md]: Fixed

Comment (langsmith) [reply src/pages/maps/overview/styling-map.md]: Fixed

**Pull request merged**